sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/datastore/EntityToRowRowToEntityTest.java

```diff
@@ -74,7 +73,7 @@
           .addNullableField("rowArray", array(FieldType.row(NESTED_ROW_SCHEMA)))
           .addNullableField("double", DOUBLE)
           .addNullableField("bytes", BYTES)
-          .addNullableField("string", CalciteUtils.CHAR)
+          .addNullableField("string", CHAR)
```

Comment:
Will this work if we just use `STRING`? (Rather than re-creating `CalciteUtils.CHAR` here)

Reply:
Done